### PR TITLE
chore: temporarily disable storage changes workflow for pull requests

### PR DESCRIPTION
**Pull request by gitcoindev**
Temporarily disable for pull requests until limiting rate API issue is resolved.

Reference: #876

### CODE REVIEW

**Review by molecula451 — APPROVED** [review]
but let's get to the root of the cause @gitcoindev, as this is one of the most sophisticated implementation we have that checks for diamond storage @rndquu

**Comment by gitcoindev** [comment]
> but let's get to the root of the cause @gitcoindev, as this is one of the most sophisticated implementation we have that checks for diamond storage

CodeQL triggered the rate limit API error, perhaps it is the root cause or combination of both: CodeQL analysis and storage changes. I disabled this one as well.

**Review by rndquu — APPROVED** [review]
I've added [here](https://github.com/ubiquity/ubiquity-dollar/issues/876) a reminder to revert the current PR

**Comment by gitcoindev** [comment]
Now coverage job is failing, coveralls server is down or API endpoint not responding. I triggered it 3 times already with the same result..

📄 Using coverage file: packages/contracts/filtered-lcov.info
🚀 Posting coverage data to https://coveralls.io/api/v1/jobs
⚠️ Internal server error. Please contact Coveralls team.
Error: Process completed with exit code 1.

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
 
 on:
-  pull_request:
   push:
+    branches:
+      - development
+  # pull_request:  ## temporarily disable for pull requests until https://github.com/ubiquity/ubiquity-dollar/issues/876 is resolved.
 
 jobs:
   code-ql-analysis:
```
Comment (0x4007) [inline .github/workflows/codeql-analysis.yml:10]: Why not disable it entirely including on push? You mentioned that Code QL analysis is likely to be an issue

**Comment by 0x4007** [comment]
Isn't there a coverage solution we can use that doesn't require third party APIs? I'm out at an event now or else I would contribute more useful research. 

https://www.perplexity.ai/search/486e5ef2-96ef-4ac8-8845-818f8bfd0f3b

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (gitcoindev) [reply .github/workflows/codeql-analysis.yml:10]: all right let's disable completely for now

**Comment by gitcoindev** [comment]
> Isn't there a coverage solution we can use that doesn't require third party APIs? I'm out at an event now or else I would contribute more useful research.
> 
> https://www.perplexity.ai/search/486e5ef2-96ef-4ac8-8845-818f8bfd0f3b

Coveralls is fixed now, this was a temporary glitch on their side. But if this happens again we could consider other options.

**Pull request merged**